src/routing/command.rs

```diff
@@ -40,6 +42,17 @@ pub(crate) enum Command {
     HandleVote { vote: Vote, proof_share: ProofShare },
     /// Handle consensus on a vote.
     HandleConsensus { vote: Vote, proof: Proof },
+    /// Handle the result of a DKG session we are participating in
+    HandleDkgParticipationResult {
+        dkg_key: DkgKey,
+        elders_info: EldersInfo,
+        result: Result<DkgOutcome, ()>,
+    },
+    /// Handle the result of a DKG session we observing
```

Comment:
according to the comment, these two entries are very similar and confusing.
maybe expand the comment bit more?